packages/trpc/server/routers/viewer/workflows/activateEventType.handler.ts

```diff
@@ -164,8 +164,9 @@ export const activateEventTypeHandler = async ({ ctx, input }: ActivateEventType
     const remindersToDelete = await prisma.workflowReminder.findMany({
       where: {
         booking: {
-          eventTypeId: eventTypeId,
-          userId: ctx.user.id,
```

Comment:
We don't need this. And this caused the issue that reminders for team event types weren't cancelled because the logged-in user wasn't the host of that booking